test/integration/test_conferences.py

```diff
@@ -0,0 +1,488 @@
+"""
+Integration tests for Bandwidth's Voice Voice Conferences API
+"""
+
+from test.utils.env_variables import *
+from test.utils.call_cleanup import callCleanup
+import json
+import time
+from typing import List, Tuple
+import unittest
+from hamcrest import assert_that, has_properties, not_none, instance_of, greater_than
+
+
+import bandwidth
+from bandwidth.api import calls_api
+from bandwidth.model.create_call import CreateCall
+from bandwidth.model.create_call_response import CreateCallResponse
+from bandwidth.model.call_state import CallState
+from bandwidth.model.call_state_enum import CallStateEnum
+from bandwidth.model.update_call import UpdateCall
+from bandwidth.model.redirect_method_enum import RedirectMethodEnum
+from bandwidth.api import conferences_api
+from bandwidth.model.conference_state_enum import ConferenceStateEnum
+from bandwidth.model.conference_recording_metadata import ConferenceRecordingMetadata
+from bandwidth.model.update_conference import UpdateConference
+from bandwidth.model.update_conference_member import UpdateConferenceMember
+from bandwidth.model.file_format_enum import FileFormatEnum
+from bandwidth.rest import RESTClientObject, RESTResponse
+from bandwidth.exceptions import ApiException, UnauthorizedException, ForbiddenException, NotFoundException
```

Comment:
Order of imports in python [according to Pep8](https://peps.python.org/pep-0008/#imports) is 

built in libraries 

3rd party installed libraries 

local modules

```
import time

import hamcrest 

import bandwidth
```

Reply:
IMO we should implement autopep8 or black to run on all PRs and autoformat code

Reply:
OK, I changed the order of imports but I think implementing something. like autopep8 should be moved to a new card